org.srb2.SRB2Persona.yaml

```diff
@@ -0,0 +1,104 @@
+app-id: org.srb2.SRB2Persona
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
```

Comment:
current runtime is 22.08

This one will be EOL in 4 month.

Reply:
Like I said [here](https://github.com/flathub/flathub/pull/4138#issuecomment-1541122350), I used the SRB2 Flatpak as a base to build this one, but I can test the latest runtime to check if it works correctly.